src/scraper/scraper.cpp

```diff
@@ -2104,15 +2105,19 @@ bool ProcessProjectRacFileByCPID(const std::string& project, const fs::path& fil
                 // The username has to be temporarily changed to the key, so that
                 // it matches the key of the mPendingMap, which is the string representation
                 // of the pending beacon CKey. This is the crux of the user validation.
-                const auto iter_pair = Consensus.mPendingMap.find(username);
+                const auto& iter_pair = Consensus.mPendingMap.find(username);
```

Comment:
`find()` returns a `std::map<...>::iterator` object by value. Should not matter here because the compiler will collapse the `const&`, but it may be worth reverting this for future clarity.

Reply:
Yeah. You are right.